src/relay/transforms/dynamic_to_static.cc

```diff
@@ -46,6 +47,14 @@ class DynamicToStaticMutator : public MixedModeMutator {
         static const Op& reshape = Op::Get("reshape");
         return Call(reshape, {call_node->args[0]}, Attrs(attrs), {});
       }
+    } else if (call_node->op == dyn_tile_op_) {
```

Comment:
Just out of curiosity, the dynamic list could be quite long, will we enumerate each case here?

Reply:
I think we'll need to, there's no other way to dispatch on individual ops types. We could do it with a switch statement for performance, or implement a registry lookup to move the code out of of this function.

Reply:
yeah, thanks, I am oaky either way. Another question, what will happen if users forget to do this conversion? Will the program crash or proceed with the dynamic op?

Reply:
As the purely dynamic ops mature, I'm thinking we'll add this pass to the defaults so it runs a a normal part of compilation. Until then, without this pass dynamic ops will run on the VM and fail on the Graph Runtime, as we have with current behavior.